dftimewolf/lib/collectors/grr.py

```diff
@@ -0,0 +1,1070 @@
+"""Collects artifacts with GRR."""
+
+import datetime
+import os
+import re
+import syslog
+import tempfile
+import time
+import zipfile
+
+from dftimewolf.lib.collectors.collectors import BaseCollector
+
+from grr_api_client import errors as grr_errors
+from grr_api_client import api as grr_api
+from grr_api_client.proto.grr.proto import flows_pb2
```

Comment:
I don't know why it's complaining, this was added to `requirements.txt`. I know that `grr_api_client` does some Python path manipulation so it could come from that.